corporate/lib/stripe.py

```diff
@@ -939,3 +943,11 @@ def update_billing_method_of_current_plan(
                 "charge_automatically": charge_automatically,
             },
         )
+
+
+def check_license_limit(realm: Realm) -> None:
+    plan = get_current_plan_by_realm(realm)
+    if plan is None or plan.automanage_licenses:
+        return
+    if plan.licenses() <= get_latest_seat_count(realm):
+        raise LicenseLimitError()
```

Comment:
Should this be called something like `check_another_seat_can_be_added`? `if plan.licenses() <= get_latest_seat_count(realm):` enforces that something can be added, which is slightly stronger than the current function name suggests

Reply:
Yeah. Makes sense. Will update.